src/test/java/bio/terra/service/dataset/flight/update/DatasetSchemaUpdateValidationTest.java

```diff
@@ -67,12 +65,12 @@ public void setup() {
                 .tables(
                     List.of(
                         new DatasetTable()
-                            .name(existingTableName)
+                            .name(EXISTING_TABLE_NAME)
                             .id(existingTableId)
                             .columns(
                                 List.of(
                                     new Column()
-                                        .name(existingColumnName)
+                                        .name("existing_column")
```

Comment:
Not sure why you made this not a constant?  now it doesn't match existing_table_name